presto-hive/src/main/java/com/facebook/presto/hive/HdfsConfigurationUpdater.java

```diff
@@ -208,6 +212,11 @@ public void updateConfiguration(PrestoHadoopConfiguration config)
         config.setLong(PrestoS3FileSystem.S3_MULTIPART_MIN_PART_SIZE, s3MultipartMinPartSize.toBytes());
         config.setBoolean(PrestoS3FileSystem.S3_PIN_CLIENT_TO_CURRENT_REGION, pinS3ClientToCurrentRegion);
         config.set(PrestoS3FileSystem.S3_USER_AGENT_PREFIX, s3UserAgentPrefix);
+
+        // set config for Azure Blob
+        if ((wasbAccessKey != null) && (wasbStorageAccount != null)) {
```

Comment:
The parentheses around each comparison are unnecessary. Equality comparisons have higher precedence that logical AND.